commit 4bbe80a68768b0714be7e6decdfa5880b8347723
Author: C Schell <kdebugs@schells.com>
Date:   Wed Oct 9 21:50:47 2013 -0700

    Fix zoom effect cursor position on initial zoom in.
    
    BUG: 325799
    FIXED-IN: 4.11.3

diff --git a/kwin/effects/zoom/zoom.cpp b/kwin/effects/zoom/zoom.cpp
index f34d707..b06583d 100644
--- a/kwin/effects/zoom/zoom.cpp
+++ b/kwin/effects/zoom/zoom.cpp
@@ -387,8 +387,9 @@ void ZoomEffect::zoomIn(double to)
         polling = true;
         effects->startMousePolling();
     }
+    cursorPoint = effects->cursorPos();
     if (mouseTracking == MouseTrackingDisabled)
-        prevPoint = effects->cursorPos();
+        prevPoint = cursorPoint;
     effects->addRepaintFull();
 }
 
